imagebuildah/build.go

```diff
@@ -73,6 +75,8 @@ type BuildOptions struct {
 	Compression archive.Compression
 	// Arguments which can be interpolated into Dockerfiles
 	Args map[string]string
+	// Whether build arg default value should be explicitly set
+	BuildArgsMandatory bool
```

Comment:
Why would a user not want this? Does Docker always fail if the BuildArgs are not specified?  If yes, then just remove the option and enforce that the buildah are is mandatory.